### ISSUE TITLE
Emmet - capital letter

### ISSUE THREAD

**Issue by user3323** [Microsoft/vscode#33028]
- VSCode Version: 1.15.1
- OS Version: win7x64

Every time you write any word starting with capital letter - Emmet pitches in process and suggests you it's abbreviation. So, if you wanna write "Some (_Enter_) Text (_Enter_)" - it's gonna be two tags, not words.

GIF:
![emmet - capital letter](https://user-images.githubusercontent.com/24613274/29616000-8cea84de-8818-11e7-9c79-a1bea9b4ec95.gif)

**Comment by ramya-rao-a** [comment]
@user3323 I am not able to repro using VS Code 1.15.1 I tried in Mac and Windows 10

Can you share your settings?
Can you also run `code --disable-extensions` and see if you can repro in the absence of other extensions?

**Comment by user3323** [comment]
@ramya-rao-a, hello. I found the reason. It's all about the setting called "**emmet.syntaxProfiles**", in particular in values of "**tag_case**" property of "html" group.

In the GIF above, the setting configuration was:

```
"emmet.syntaxProfiles": {
  "html": {
    "tag_case": "lower"
  }
}
```
Very strange behavior. This setting has to be responsible for absolutely another thing. So, when it's set to "lower" it will suggest you to expand snippet after any word started from capital letter. And when it's set to "upper" - it will suggest after any word started from lower case letter. And the only value "asis" - doesn't cause any problems.

Here is another GIF, that shows everything:
![emmet syntaxprofiles - capital letter](https://user-images.githubusercontent.com/24613274/29649541-587b4d20-889d-11e7-98a6-9eef3051392b.gif)

PS: also by testing this "tag_case" setting and trying to expand simple "div", I found that not all three values of this setting actually work, as they should. Gonna make a new issue.
